### PR TITLE
fix USD amount and UI

### LINKED ISSUE #152: Display of USD amount is incorrect in mobile mode
https://yearn.watch/vault/0x7047F90229a057C13BF847C0744D646CFb6c9E1A

Desktop display:
Screen Shot 2021-09-17 at 09 18 31

Mobile Display:
Screen Shot 2021-09-17 at 09 18 44

### PR DESCRIPTION
**Pull request by laitanop**
Screen Shot 2021-09-19 at 16 35 27
Screen Shot 2021-09-19 at 16 35 18

### CODE REVIEW

File: src/components/common/ProgressBar/index.tsx [hunk 1]
```diff
@@ -37,10 +42,7 @@ const ProgressBars = (props: any) => {
                 <Box width="100%" mr={1}>
                     <BorderLinearProgress variant="determinate" value={f} />
                 </Box>
-                <Typography
-                    variant="body2"
-                    color="textSecondary"
```
Comment (storming0x) [inline src/components/common/ProgressBar/index.tsx:42]: is this change needed for this issue?

File: src/components/app/Home/index.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ import {
 
 import { GlobalStylesLoading } from '../../theme/globalStyles';
 import { Container } from '@material-ui/core';
-
+import backgroundLoading from '../../../images/mitch.svg';
```
Comment (storming0x) [inline src/components/app/Home/index.tsx]: is this change needed for this fix?

File: src/components/theme/Themes.js [hunk 3]
```diff
@@ -1,7 +1,6 @@
-// import backgroundImageLight from '../../images/background_image_yearn_watch_light.svg';
 import backgroundImageDark from '../../images/background_image_yearn_watch_dark.jpeg';
-import backgroundImageLightNoMitch from '../../images/yearn_background_light_no_mitch.svg';
-import backgroundLoading from '../../images/loadingBackground1.svg';
+import backgroundImageLightNoMitch from '../../images/yearn_background_light_no_mitch.png';
```
Comment (storming0x) [inline src/components/theme/Themes.js]: PNG files of 2MB are too big for web, they need to be optimized

**Review by storming0x — CHANGES_REQUESTED** [review]
Should remove every other change, except the display fix of the amount

File: src/components/common/ProgressBar/index.tsx (hunk 1 above)
Comment (laitanop) [reply src/components/common/ProgressBar/index.tsx:42]: yes.

File: src/components/app/Home/index.tsx (hunk 2 above)
Comment (laitanop) [reply src/components/app/Home/index.tsx]: Deleted image

File: src/components/theme/Themes.js (hunk 3 above)
Comment (laitanop) [reply src/components/theme/Themes.js]: Deleted image png

**Review by storming0x — APPROVED** [review]

**Pull request merged**